### DISCUSSION TITLE
Lunaline background color right under window separator of Nvim-Tree

### DISCUSSION THREAD

**Discussion by MKrbm** [ayamir/nvimdots#961] [Q&A] [answered]
![image](https://github.com/ayamir/nvimdots/assets/57752203/757f0b8f-f238-4f63-884b-30a65f64ef15)
How can I set surface0 instead of mantle color here?

**Comment by Jint-lzxy** [comment]
`:h hl-StatusLine`:
```console
							*hl-StatusLine*
StatusLine	Status line of current window.
```

**Comment by MKrbm** [comment]
When I focus on nvim-tree, the color will set on the target part of Lualine. But when I focus on the editor, then not.

When editor window is focused
![image](https://github.com/ayamir/nvimdots/assets/57752203/c304384f-a953-4eb0-800c-b2392967f61c)

When nvim-tree is focused
![image](https://github.com/ayamir/nvimdots/assets/57752203/5b801502-eaf3-4de3-9a5f-cde77356cfa5)

**Reply by Jint-lzxy** [reply]
For filetype-specific extensions/highlights, u need to modify lualine's components directly:
https://github.com/ayamir/nvimdots/blob/8631738b9de4a5477fbf4387e4822bafd6716f2f/lua/modules/configs/ui/lualine.lua#L20-L48

**Comment by MKrbm** [comment]
I think I've tried every single potential variables in the file but nothing affected this part

![image](https://github.com/ayamir/nvimdots/assets/57752203/83db23a8-f41c-486c-a278-bb3355855be9)

**Reply by Jint-lzxy** [reply]
Oh I see what u mean. In fact, this has nothing to do with lualine, it's:
```console
							*hl-WinSeparator*
WinSeparator	Separators between window splits.
```

**Reply by MKrbm** [reply]
No, it's not effective unfortunately. 
Maybe my explanation was unclear 

I'd like to change the color of this area

![image](https://github.com/ayamir/nvimdots/assets/57752203/66331f61-d4b0-4e6f-8e6e-0de8428ce924)

The closest one was [second picture of last post ](https://user-images.githubusercontent.com/57752203/262217460-5b801502-eaf3-4de3-9a5f-cde77356cfa5.png). But this one is not enough since this color is only applied when nvimtree is focused.

Thank you and sorry for taking your time.

**Reply by Jint-lzxy** [reply, marked as answer]
This looks like the Normal background of nvim-tree itself. See :h nvim-tree-highlight
